src/main/java/ServiceLayer/SystemService.java

```diff
@@ -0,0 +1,71 @@
+package ServiceLayer;
+import java.util.logging.Logger;
+import java.util.logging.Level;
+import Domain.ExternalServices.ExternalServiceHandler;
+
+// Class that represents the system service and enables users (probably admins) to control the system.
+
+public class SystemService {
+     private UserService userService;
+    private ExternalServiceHandler externalServiceHandler;
+    private boolean isOpen = false;
+    private static final Logger logger = Logger.getLogger(SystemService.class.getName());
+
+    public SystemService(UserService userService, ExternalServiceHandler externalServiceHandler) {
+        this.userService = userService;
+        this.externalServiceHandler = externalServiceHandler;
+    }
+   
+    /**
+     * Opens the system.
+     * 
+     * @param userId   the user ID
+     * @param password the user password
+     * @return a response indicating the success or failure of opening the system
+     */
+    public Response openSystem(String userId, String password) {
+        Response response = new Response();
+        try {
+            // Perform user login
+            Response loginResponse = userService.logIn(userId, password);
```

Comment:
i think its happend after he does login, maybe just check he is logged in in boolean in user class